Apps/WebClient/test/functional/e2e/cypress/integration/user/feedback.js

```diff
@@ -0,0 +1,29 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('User Feedback', () => {
+    beforeEach(() => {
+        cy.login(Cypress.env('keycloak.username'), 
+                 Cypress.env('keycloak.password'), 
+                 AuthMethod.KeyCloak)
+    })
+
+    it('Send feedback', () => {
+        cy.get('[data-testid=feedbackContainer]').click()
+        cy.get('[data-testid=feedbackCommentInput]')
+            .type('Great job team!');
+        cy.get('[data-testid=sendFeedbackMessageBtn]').click()
+        cy.get('.fa-check-circle').should('be.visible');
```

Comment:
Why the class selector?

Reply:
same it is a bootstrap component, but I think we don't have to check this as the 2 lines below it should validate the test

Reply:
removed the .fa-check-circle